src/back/spv/writer.rs

```diff
@@ -742,51 +865,33 @@ impl Writer {
 
         let instruction;
 
-        match ty.inner {
+        match ty.inner.clone() {
```

Comment:
We shouldn't need cloning here. What was the problem that made you clone?

Reply:
True, I think I got an intellisense error that `TypeInner` could not be moved, but when I remove it now, the "error" is gone. Probably a caching bug in my IDE.